tests/testthat/test-live.R

```diff
@@ -0,0 +1,26 @@
+context("live scenario test")
+test_that("Scenario Test", {
+  testthat::skip_on_travis()
+  credentialsFileName <- "credentials.json"
+  clusterFileName <- "cluster.json"
+
+  doAzureParallel::generateCredentialsConfig(credentialsFileName)
+  doAzureParallel::generateClusterConfig(clusterFileName)
+
+  doAzureParallel::setCredentials(credentialsFileName)
```

Comment:
What does this auth against?

Reply:
The user will just use their own auth. I will add comments.